packages/traceability/issuer-api/migrations/1622459627494-RmCertificationRequestStatus.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RmCertificationRequestStatus1622459627494 implements MigrationInterface {
+    name = 'RmCertificationRequestStatus1622459627494';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "issuer_certification_request" DROP COLUMN "status"`);
```

Comment:
This is some additional change?

Reply:
Yup. Certification requests no longer need the `status` field because they are stored only once they are deployed to the blockchain.